### PR TITLE
Updating documentation README.md 

### PR DESCRIPTION
**Pull request by pgerhardt**
README.md now clarifies what this repo is for and points to N2NP rfc document.

### CODE REVIEW

**Comment by 0x5c** [comment]
The "working" branch is `work/louis/routing`. Comments should be made on pr #2.
Also clarifying the repo's purpose might make the project too much public, something that @N-O-D-E asked to avoid for now.

**Review by pgerhardt — COMMENTED** [review]
Updated

**Comment by 0x5c** [comment]
Add the branch name
"See `./protocols/N2NP.rfc` in branch `work/louis/routing` to review the[...]"

**Comment by 0x5c** [comment]
~~And remove the strikethrough.~~
*EDIT:* Done

**Comment by pgerhardt** [comment]
Updated again

**Comment by pgerhardt** [comment]
Merged changes

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Documentation
 
+Documentation for the [redacted]  project.
```
Comment (Ne02ptzero) [inline README.md]: ```[redacted]``` ? Otherwise, Looks good to me 👍

Comment (pgerhardt) [reply README.md]: `[redacted]` is weird to me too.  Per @0x5c in slack. Happy to change it back if you (@Ne02ptzero) are cool with it.

**Comment by pgerhardt** [comment]
Updated @Ne02ptzero

File: README.md (hunk 1 above)
Comment (Ne02ptzero) [reply README.md]: @0x5c ?

**Pull request merged**